### PR TITLE
[16.0][IMP] apriori: initial filling

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
...

### CODE REVIEW

File: openupgrade_scripts/apriori.py [hunk 1]
```diff
@@ -5,21 +5,56 @@
...
+    "stock.location.route": "stock.route",
+    "stock.production.lot": "stock.lot",
     # OCA/...
 }
 
 # only used here for upgrade_analysis
-merged_models = {}
+merged_models = {
+    # odoo
+    "account.account.type": "account.account",
```
Comment (pedrobaeza) [inline openupgrade_scripts/apriori.py]: I don't think this can be considered a merge...

Comment (MiquelRForgeFlow) [reply openupgrade_scripts/apriori.py]: But it's a merge in a sense, right? Ok, I will remove it.

Comment (legalsylvain) [reply openupgrade_scripts/apriori.py]: indeed, in a sense, but here merged models is used by the module_analysis to understand if fields are new or not or moved on a model. writing that will generate potential bad analysis on ``account.account`` model I think.

**Review by legalsylvain — COMMENTED** [review]
- PR ok for me. @pedrobaeza what do you think ? Could be merge to allow other PR that depends on the initial one.

- do you think that we should push in the same time, the update of the analysis. (analysis should change, due to renamings / mergings...)

**Pull request merged**